Makefile

```diff
@@ -5,8 +5,9 @@ CFLAGS := -W -Wall -Werror -std=gnu9x
 
 BINDIR := /usr/bin
 LIBEXECDIR := /usr/libexec
+MANDIR := /usr/share/man
 
-.PHONY: build install check clean
+.PHONY: build install check clean docs
 
 build: clrtrust-helper clrtrust
```

Comment:
I'm sorry if I confused you. It was either not using `ronn` or remove it from the default target, but not both. The reason for this is simple: we need to run this build in mock environment for package and should be able to build it.

A really proper way to define `build` target would actually be along these lines:

```
build: clrtrust-helper clrtrust man/clrtrust.1

man/%.1: man/%.1.md
       pandoc -s -f markdown -t man $< --output $@

clean:
       rm -rf clrtrust-helper clrtrust-helper.o clrtrust man/clrtrust.1
```

And no need to have `docs` as a phony target.

Additionally, the `.gitignore` needs to be modified to ignore all the files in `man/` directory, except for `.md` files:

```
man/
!man/*.md
```

Does it make sense?